boards/nrf9160dk/include/periph_conf.h

```diff
@@ -66,13 +66,13 @@ static const spi_conf_t spi_config[] = {
 static const timer_conf_t timer_config[] = {
     {
         .dev      = NRF_TIMER0_S,
-        .channels = 3,
+        .channels = 5,
```

Comment:
```suggestion
        /* using first channel for timer_read(), so only 5 of 6 channels available */
        .channels = 5,
```

Reply:
That's the case here. nRF9160/nRF53 has 6 capture/compare channels per timer.
 Looking at the driver, the last one is reserved to do the capture task of the current counter. While the remaining are available to do the compare tasks. I agree that this lacks a proper documentation unless you look at the code to try to understand what's going on.
The question is: Where is the best place to add this ? Within the board configuration in `periph_conf.h` ? Or in the driver itself ?

Reply:
Just looked at the first comment without reading the rest. Ignore this :)